Felienne/hedy
# Adds parsons step to tutorial

**Description**
In this PR we add a step for the Parsons to the tutorial. We only add the translation and structure in en/nl YAML files, Weblate should take care of the rest. **Do not deploy until Weblate is up-to-date as well.**


**Fixes**
This PR fixes #3171.

**How to test**
Start the tutorial mode by navigating to `/tutorial`. Verify that there is new step for the Parsons problems while everything still works expected. Also we add a minor improvement where the quiz tab is selected on the corresponding step.
